internal/rbd/rbd_util.go

```diff
@@ -2155,3 +2156,24 @@ func (rv *rbdVolume) unsetAllMetadata(keys []string) error {
 
 	return nil
 }
+
+// getVolumeContext filters out parameters that are not required in the volume context.
+func (rv *rbdVolume) getVolumeContext(parameters map[string]string) map[string]string {
```

Comment:
we dont need to make this as a method, moving this to outside util is good and we can pass the notRequiredParams so that we can reuse this same function for cephfs as well.

Reply:
@Madhu-1 
will move this to `internal/util/util.go`  and rename as `RemoveParameters(params, notRequiredParams)` ?

Reply:
Yes that also helps